onyo/lib/onyo.py

```diff
@@ -92,20 +92,22 @@ def clear_caches(self) -> None:
         self._gitfiles = None
         self._templates = None
 
-    def generate_commit_message(self, message: str = "", cmd: str = "",
+    def generate_commit_message(self, message: list[str] = [], cmd: str = "",
                                 keys: list[str] = [], destination: str = "") -> str:
         """
-        Returns either the commit message given to onyo by the user, or creates
-        a default commit message.
+        The commit message is generated based on arguments for `--message` if
+        given by the user, or a default commit message.
+        Adds a list of changed files to the body of commit messages.
         """
         message_header = ""
+        message_body = ""
         MAX_LEN = 80
+        assets = [Path(self.opdir, x).relative_to(self.root) for x in sorted(self.files_staged)]
 
-        # TODO: first construct body?
         if message:
-            message_header = message
+            message_header = "".join(message[0])
```

Comment:
This can be simply:
```
message_header = message[0]
```

Reply:
It can't, because the way argparser does work when doing `action='append'`.
If it is done like`message_subject = message[0]`, the commit message looks like ['word'], e.g. it is still an array.
But instead of `join()`ing them, I changed it now to `message_subject = message[0][0]`, and for the body similarly.